internal/cmd/run.go

```diff
@@ -287,6 +290,7 @@ func runCmd() *cobra.Command {
 	cmd.Flags().StringArrayVarP(&replaceScripts, "replace", "r", nil, "Replace instructions using sed.")
 	cmd.Flags().BoolVarP(&parallel, "parallel", "p", false, "Run tasks in parallel.")
 	cmd.Flags().BoolVarP(&runAll, "all", "a", false, "Run all commands.")
+	cmd.Flags().BoolVarP(&skipPromptVars, "skip-prompt-vars", "v", false, "Skip prompting for variables.")
```

Comment:
@sourishkrout I am not sure what alias should we use for this right now `v`

Reply:
`v` is very common for `verbose` so let's not use it, please. How about we rename it to `--skip-prompts` with `s` as alias the `vars` part is redundant.

Reply:
we are removing the alias only `--skip-prompts` will be available.